examples/mytask.rb

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
+
+require_relative "../lib/task_helper.rb"
+
+class MyClass < TaskHelper
+  def self.task(params = {})
+    {'result': "Hi, my name is #{params['name']}"}
```

Comment:
quotes are misleading here. this will actually be a symbol so it may as well be `{result: "Hi..."}`

I think the result key may also be a bit confusing and result in users always adding it {greeting: "Hi..."} seems slightly less confusing